docs/importing-data.md

```diff
@@ -2,17 +2,17 @@
 
 ## Importing Data
 
-Data may be imported from Google Sheets or CSV files into SQL tables. Columns in these spreedsheets must be named using type suffixes.
+Data may be imported from Google Sheets into SQL tables. Columns in these spreedsheets must be named using type suffixes.
 
 ### Importing Google Sheets
 
-1. Create a sheet in the "Pegasus" Google Docs folder, naming your columns using type suffixes.
-1. In the Dropbox folder for a site, create a file named `data/table_name.gsheet` where table_name is the name you'd like the database table to have. The file should contain the "Pegasus" Google Docs path to the sheet you created in step 1. For example, `code.org/data/about_team.gsheet` would contain `Data/About/Team` and the data will be imported into a table named `about_team`.
-1. The data should import on staging automatically within a few minutes.
-
-### Importing CSV files
```

Comment:
This file was last updated 10 years ago, and I don't think importing CSV files is relevant anymore.

Reply:
@bethanyaconnor is this something you might know about? No worries if not!

Reply:
Not 100% sure but I believe we only import CSV files from google sheets now. I think we looked and the `data/` folder doesn't exist right? So either way, this documentation isn't useful, so I'd remove it. We can add documentation if we do discover this

Reply:
Ok cool, ty!

Reply:
I think we also import [CSV files from AWS](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/models/census/census_summary.rb#L43) too! (Not that this changes if this documentation should be deleted)